account/list_servers_minimal_rpc.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package account;
+
+option go_package = "./;grpc_account";
+
+message ServerMinimal {
+	string environment = 1;
+	string id = 2;
```

Comment:
What is this ID?

Reply:
A unique ID given to the server once it's been created in the database, so it can be referenced. It's exact value is an implementation detail